flocker/restapi/test/test_infrastructure.py

```diff
@@ -412,17 +418,20 @@ def test_decode(self, logger):
         render(app.app.resource(), request)
         self.assertEqual(objects, app.kwargs)
 
-    def assertNoDecodeLogged(self, logger, method):
+    def assertNoDecodeLogged(self, logger, method, path=b"/foo/bar"):
         """
         The I{JSON}-encoded request body is ignored when the given method is
```

Comment:
Next issue in queue is "don't require application/json", so can add a test there for another ignore_body variant.